File: Algorithms/01 Warmup/C#/03 Compare the Triplets/CompareTriplets.cs
using System;
using System.Collections.Generic;

class Solution {

    static List<int> compareTriplets(List<int> a, List<int> b) {
        List<int> response = new List<int>() { 0,0};
        for(int i = 0; i < a.Count; i++) {
            if(a[i] > b[i])
                response[0] += 1;
            else if(a[i] < b[i])
                response[1] += 1;
        }
        return response;

    }

    static void Main(String[] args) {

        List<int> a = new  List<int> { 5, 3, 5 };
        List<int> b = new List<int> { 1, 5, 2 };

        Console.WriteLine(String.Join(" ",compareTriplets(a,b)));
        Console.ReadKey();
    }
}

File: Algorithms/01 Warmup/Javascript/03 Compare the Triplets/CompareTriplets.js
'use strict';

// Complete the compareTriplets function below.
function compareTriplets(a, b) {
    var solution = []
    solution[0] = 0;
    solution[1] = 0;
    for (let i = 0; i < a.length; i++) {
        if (a[i] > b[i])
            solution[0]++;
        else if (a[i] < b[i])
            solution[1]++;
    }
    return solution;

}


function main() {

    const a = [10,20,30];
    const b = [5,25,29];

    const result = compareTriplets(a, b);
    console.log(result);
}
main();